package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@livekit/components-react": "1.3.0",
+    "@livekit/components-core": "file://../components/packages/core",
+    "@livekit/components-react": "file://../components/packages/react",
```

Comment:
local packages?

Reply:
when publishing those will be automatically replaced by pnpm https://pnpm.io/workspaces#referencing-workspace-packages-through-aliases

Reply:
🤦 thanks!